package.json

```diff
@@ -5,8 +5,8 @@
   "main": "plugin.js",
   "types": "plugin.d.ts",
   "scripts": {
-    "test": "tap 'test/**/*.test.js'",
-    "test-ci": "tap --cov 'test/**/*.test.js'",
+    "test": "tap test/**/*.test.js && npm run typescript",
```

Comment:
This line will fail on my system.

Reply:
The old way fails on windows (executes 0 tests)

Reply:
I think this should just be `tap test/*test.js`

Reply:
@jsumners You mean the test fails or the removal of `'` ?

Reply:
I mean I do not have the `typescript` binary in my path. So if it isn’t a direct dependency of the project then it will not be found and an error will occur.

Reply:
it's in the devDependencies, it will be picked up automatically by npm.

Reply:
`devDependencies` have `typescript` should it be `ts-node` instead?

Reply:
Nope. If it's in the `devDependencies` then it's all good.